cmd/main_android.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"log"
+
+	"github.com/getlantern/osversion"
+
+	_ "golang.org/x/mobile/app"
+)
+
+func main() {
+	str, err := osversion.GetHumanReadable()
+	if err != nil {
+		log.Printf("Error in osversion.GetHumanReadable: %v", err)
+	}
+	// PINEAPPLE is just to make grepping through logcat easier
+	log.Printf("PINEAPPLE: %s\n", str)
```

Comment:
did you leave this in on purpose?

Reply:
Yep. See the README. This is for testing: https://github.com/getlantern/osversion/blob/9f92664c68c7aecec6efcb3ccb3d0e8856ce5289/README.md#L36

It's just an [egg](https://shellcode.blog/Windows-Exploitation-Egg-hunting/)